samples/core/xgboost_training_cm/README.md

```diff
@@ -12,6 +12,8 @@ or not.
 
 ## Requirements
 
+> :warning: If you are using **full-scope** cluster in hosted pipeline beta version, **DO NOT** follow this section.
+
```

Comment:
even in full-scope, we still need to enable API

Reply:
Done.